addons/ofxEmscripten/src/ofxAppEmscriptenWindow.cpp

```diff
@@ -359,27 +271,26 @@ void ofxAppEmscriptenWindow::disableSetupScreen(){
 	bEnableSetupScreen = false;
 }
 
-void ofxAppEmscriptenWindow::setVerticalSync(bool enabled){
-	eglSwapInterval(display, enabled ? 1 : 0);
-}
 
-EGLDisplay ofxAppEmscriptenWindow::getEGLDisplay(){
-	return display;
+ofCoreEvents & ofxAppEmscriptenWindow::events(){
+	return _events;
 }
 
-
-EGLContext ofxAppEmscriptenWindow::getEGLContext(){
-	return context;
+shared_ptr<ofBaseRenderer> & ofxAppEmscriptenWindow::renderer(){
+	return _renderer;
 }
```

Comment:
Curious if all this stuff needs to be removed? 
Do we not have a context anymore?

Reply:
most of the changes in this file are from @roymacdonald : https://github.com/openframeworks/openFrameworks/issues/6781#issuecomment-974879862
as far as I can tell it is a web_gl context now.